src/pages/portfolio/Portfolio.tsx

```diff
@@ -119,13 +123,27 @@ export default () => {
                     {
                       value: PortfolioRoute.Positions,
                       slotBefore: <Styled.Icon iconName={IconName.Positions} />,
-                      label: stringGetter({ key: STRING_KEYS.POSITIONS }),
+                      label: (
+                        <>
+                          {stringGetter({ key: STRING_KEYS.POSITIONS })}
+                          {numPositions > 0 && (
+                            <Tag type={TagType.Number}> {numPositions} </Tag>
+                          )}
+                        </>
+                      ),
                       href: PortfolioRoute.Positions,
                     },
                     {
                       value: PortfolioRoute.Orders,
                       slotBefore: <Styled.Icon iconName={IconName.OrderPending} />,
-                      label: stringGetter({ key: STRING_KEYS.ORDERS }),
+                      label: (
+                        <>
+                          {stringGetter({ key: STRING_KEYS.ORDERS })}
+                          {numOrders > 0 && (
```

Comment:
can use `numOrders && (...` since 0 is falsy

Reply:
hmm I might be misunderstanding - but I think this'll end up returning `0 && ...` which will evaluate to 0 and end up rendering a 0 on screen

Reply:
nvm ur right 😛